src/LaravelRequestDocs.php

```diff
@@ -226,15 +247,27 @@ public function getControllersInfo(array $onlyMethods): Collection
      */
     public function appendRequestRules(Collection $docs): Collection
     {
+        /** @var Doc $doc */
         foreach ($docs as $doc) {
             if ($doc->isClosure()) {
                 // Skip to next if controller is not exists.
                 continue;
             }
 
-            $controllerReflectionMethod = new ReflectionMethod($doc->getControllerFullPath(), $doc->getMethod());
+            try {
+                $controllerReflectionMethod = new ReflectionMethod($doc->getControllerFullPath(), $doc->getMethod());
```

Comment:
When using an `apiResource` or a `resource` in the routes and if any method is not found an exception is generated, to prevent this, try catch was used. I'm going to change it so that the try covers all the code within the foreach, this way I believe it will be more comprehensive
![image](https://github.com/rakutentech/laravel-request-docs/assets/38543704/479c5af6-8144-4bc6-b4a3-ce649d0ec841)